pkg/eval/math/math.go

```diff
@@ -0,0 +1,286 @@
+// Package math exposes functionality from Go's math package as an elvish
+// module.
+package math
+
+import (
+	"math"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var e
+//
+// ```elvish
+// $math:e
+// ```
+//
+// The value of E: 2.718281.... This variable is read-only.
+
+//elvdoc:var pi
+//
+// ```elvish
+// $math:pi
+// ```
+//
+// The value of Pi: 3.141592.... This variable is read-only.
```

Comment:
I'd write it as *π*, using the Greek letter in italic, and link to Wikipedia.

Reply:
Done